src/symbols.js

```diff
@@ -736,6 +736,16 @@ var symbols = {
         },
 
         // AMS Misc
+        "\\#": {
+          font: "ams",
```

Comment:
These should actually be `font: "main"` I believe – you'll notice on the test page (`make serve`) that with that change, these look like they match the other font (instead of falling back to Helvetica) and don't produce warnings in the console:

![image](https://cloud.githubusercontent.com/assets/6820/6900064/f9cc8980-d6b8-11e4-93b5-a4aa18976a6c.png)